precompiles/outposts/stride/tx.go

```diff
@@ -54,22 +54,27 @@ func (p Precompile) LiquidStake(
 		return nil, err
 	}
 
-	bondDenom := p.stakingKeeper.BondDenom(ctx)
+	// WEVMOS address is the only supported token for liquid staking
+	if token != p.WevmosAddress {
+		return nil, fmt.Errorf(ErrUnsupportedToken, token, p.WevmosAddress)
+	}
 
-	tokenPairID := p.erc20Keeper.GetDenomMap(ctx, bondDenom)
+	// TODO: Uncomment this once we register WEVMOS pair with a precompile
+	// tokenPairID := p.erc20Keeper.GetDenomMap(ctx, bondDenom)
 
-	tokenPair, found := p.erc20Keeper.GetTokenPair(ctx, tokenPairID)
+	// tokenPair, found := p.erc20Keeper.GetTokenPair(ctx, tokenPairID)
 	// NOTE this should always exist
-	if !found {
-		return nil, fmt.Errorf(ErrTokenPairNotFound, tokenPairID)
-	}
+	// if !found {
+	//	return nil, fmt.Errorf(ErrTokenPairNotFound, tokenPairID)
+	// }
```

Comment:
Do we still need this now that we have the wevmos address as field in the precompile struct?